torch/linalg/__init__.py

```diff
@@ -1626,6 +1626,15 @@
 
 The parameter :attr:`full_matrices` chooses between the full (default) and reduced SVD.
 
+The `driver` kwarg may be used in CUDA with a cuSOLVER backend to choose the algorithm used to compute the SVD.
+The choice of a driver is a trade-off between accuracy and speed.
+
+- If :attr:`A` is well-conditioned (its `condition number`_ is not too large), or you do not mind some precision loss.
```

Comment:
This needs proper indentation in lines 2, 3 and 5 as otherwise:
https://docs-preview.pytorch.org/74521/generated/torch.linalg.svd.html